### PR TITLE
Dynamically generate achievement popups

### LINKED ISSUE #128: Dynamically generate achievement popups instead of storing the whole popup as an image
We should dynamically generate achievement popups instead of storing the whole popup as an image.

### PR DESCRIPTION
**Pull request by thearst3rd**
Closes #128

### CODE REVIEW

File: tools/cmake/marbleblast.cmake
Comment (HumanGamer) [inline tools/cmake/marbleblast.cmake:1]: This change is unrelated, and could cause issues with Clang. Please remove this change.

File: engine/source/gui/game/guiAchievementPopupCtrl.cpp [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+    if (mIconTextureObject)
+    {
+        GFX->clearBitmapModulation();
+
+        RectI rect(offset.x + ACHIEVEMENT_ICON_OFFSET_X, offset.y + ACHIEVEMENT_ICON_OFFSET_Y, 64, 64);
+        GFX->drawBitmapStretch(mIconTextureObject, rect);
+    }
+
+    // I can't quite find an easy way to change the font size within this method, so both the achievement unlocked and
```
Comment (HumanGamer) [inline engine/source/gui/game/guiAchievementPopupCtrl.cpp:106]: While I would like this in the PR, if you can't figure it out, then we can deal with it later and make an issue for it.

File: game/common/local/englishStrings.inf
Comment (HumanGamer) [inline game/common/local/englishStrings.inf:1]: I would prefer if all the localization stuff related to the DLC would go in the _U1 versions of the files to keep things consistent. Can you change that?

**Review by HumanGamer — CHANGES_REQUESTED** [review]
It looks good so far, but I need you to make some changes. I've commented on those in specific.

File: game/common/local/englishStrings.inf
Comment (thearst3rd) [reply game/common/local/englishStrings.inf:1]: Ahh didn't notice those files. Yep that's an easy fix

File: engine/source/gui/game/guiAchievementPopupCtrl.cpp (hunk 1 above)
Comment (thearst3rd) [reply engine/source/gui/game/guiAchievementPopupCtrl.cpp:106]: I might just be dumb cuz I still couldn't figure it out last time but I'll look into it a bit more
